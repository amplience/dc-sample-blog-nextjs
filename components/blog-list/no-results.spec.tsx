/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import NoResults from './no-results';

describe('NoResults', () => {
  it('should render component with query', async () => {
    const component = await renderer.create(<NoResults query="search query"/>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
