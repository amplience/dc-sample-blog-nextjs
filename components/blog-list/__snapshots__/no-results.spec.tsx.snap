// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults should render component with query 1`] = `
<div
  className="jsx-2093425618 no-results"
>
  <p
    className="jsx-2093425618"
  >
    No results matching "
    search query
    "
  </p>
</div>
`;
